### ISSUE TITLE
I can not find the Playlist section.

### ISSUE THREAD

**Issue by ricardo404** [th-ch/youtube-music#2]
Hey buddy th-ch, is it possible to access the Playlist section? I chose some songs and put them in a Playlist that I created, but I can not see the place where it gives access to the created Playlist. Is there any command or something?

**Comment by th-ch** [comment]
Hey @ricardo404, when logged in, if you go in the "Library" menu (at the top), there should be a "Playlist" tab where you can find your playlists, can you find yours?

**Comment by ricardo404** [comment]
Hey th-ch friend, thank you for your response. The only options that is shown for min is "Home" and "Hotlist" without your "Library" tip. But I noticed something !, when I open the program for the first time, the "Library" section appears and then disappears from the top menu. Is it something familiar to you? Is it some blockade of YouTube itself for my region ?, which in this case is Brazil. I tried to change the region internally in the settings. I put America, and it's the same result. The "library" section appears and then disappears from the top menu, leaving only "Home" and "Hotlist". Thank you.

![Screenshot_1](https://user-images.githubusercontent.com/35298519/58770621-706fd580-8586-11e9-8c83-7beea952086b.png)

**Comment by th-ch** [comment]
> the "Library" section appears and then disappears from the top menu

The `no-google-login` plugin might be responsible for this, can you try to disable it to see if it solves your issue? (To disable it, go to the `Plugins` menu, you will then need to restart the app once it is disabled)

**Comment by ricardo404** [comment]
Could you show me how to access this menu? Because I swear to you that I can not visually find this "PLUGIN" menu. Does it get inside the same application, some submenu hidden in the application? need to give some command via keyboard? Sorry for not finding this Menu directly. I tried searching in the settings menu but I believe that the settings menu is related to the youtube player. It only has something like language, restriction, region ... Pluguin no.

**Comment by th-ch** [comment]
Indeed, frame and titlebar were hidden so the plugins menu was only available on MacOS. I pushed a new version ([1.1.4](https://github.com/th-ch/youtube-music/releases/tag/v1.1.4)) which sets the frame and titlebar when not on MacOS (so that the plugins menu is available) + removed `no-google-login` from default plugins.

It should look like this on Linux:
![linux](https://user-images.githubusercontent.com/8024624/59147235-67a15880-89f9-11e9-929d-b43a9106fc64.png)

Can you update and tell me if you get the plugins menu? If you have it, can you try to disable the `no-google-login` plugin (you will need to restart the app) to check if the Library link is back?

**Comment by ricardo404** [comment]
Very good friend th-ch, the problem was solved and I got access normally to "Library" and "Playlist" with all the songs that I had added to them. It really was the "Google No Login" that was prevented access to "Library." One thing, when I reinstalled the application, it was still listed as Google No Loguin marked as default when I went to the "Pluguin" tab. It might be some cache of mine, I believe. I think we can now mark the matter as settled. And I am willing to test future release versions of your development.
![Screenshot_2](https://user-images.githubusercontent.com/35298519/59147493-88a28300-89d2-11e9-9478-9c06a61e86b0.png)
